.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.1
+orbs:
+  node: circleci/node@2.0.1
+jobs:
+  build-and-test:
+    executor:
+      name: node/default
+      tag: "12.16"
+    steps:
+      - checkout
+      - node/install-packages
+      - run: npm install
+      - run: npx run lint
```

Comment:
Should this be `npm run lint`? I don't know if the `npx` command does anything differently here or if it is functionally the same.

Reply:
Yes it should be, thank you!